static/panes/compiler.ts

```diff
@@ -1939,7 +1939,10 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
                             const emuwindow = unwrap(jsbeebemuframe.contentWindow);
                             const tmstr = Date.now();
                             emuwindow.location =
-                                'https://bbc.godbolt.org/?' + tmstr + '#embed&autoboot&disc1=b64data:' + bbcdiskimage;
+                                'https://bbc.compiler-explorer.com/?' +
```

Comment:
(retry) this should stay `bbc.godbolt.org`

Reply:
@partouf https://bbc.compiler.explorer.com seems working. Am I missing something?

Reply:
ohhh. right